docs/source/index.rst

```diff
@@ -5,23 +5,33 @@
 
 ``labscript`` documentation
```

Comment:
Alternatively, `labscript Documentation` (regular font style for project and/or capitalise Documentation)?

Reply:
Yep. This was left over from the original Sphinx docs I started circa 2015. It's now updated to match the other docs.